src/app/components/header/header.component.html

```diff
@@ -4,15 +4,26 @@
   <div
     class="user-container h-full w-full flex justify-start items-center gap-2"
   >
+    @if ((loggedInUserInfo$ | async)?.name !== "") {
+      <p
+        class="bg-seasalt w-fit h-fit rounded-3xl px-3 py-1.5 text-xl font-pacifico text-pale-brown"
+      >
+        {{ (loggedInUserInfo$ | async)?.name?.charAt(0) }}
+      </p>
+      <p class="font-tilt-neon font-semibold text-lg text-pale-brown">
+        Hi {{ (loggedInUserInfo$ | async)?.name }}
+      </p>
+    } @else {
+      <p
+        class="bg-seasalt w-fit h-fit rounded-3xl px-3 py-1.5 text-xl font-pacifico text-pale-brown"
+      >
+        {{ currentUserName ? currentUserName.charAt(0) : "" }}
+      </p>
+      <p class="font-tilt-neon font-semibold text-lg text-pale-brown">
+        Hi {{ currentUserName ? currentUserName : "" }}
```

Comment:
You can shorten this to `currentUserName ?? ""`

Reply:
Thank you, I will do this in a future branch.